jest.config.ts

```diff
@@ -147,7 +147,6 @@ export default {
 
   // The glob patterns Jest uses to detect test files
   testMatch: [
-    "**/__tests__/**/*.[jt]s?(x)",
```

Comment:
removes the regex matcher that tests the whole `__tests__` dir so that Jest doesn't confuse with non test file or dirs